pkg/storage/types/dv.go

```diff
@@ -193,6 +193,32 @@ func HasDataVolumeErrors(namespace string, volumes []virtv1.Volume, dataVolumeSt
 	return nil
 }
 
+func HasDataVolumeExceededQuota(namespace string, volumes []virtv1.Volume, dataVolumeStore cache.Store) error {
```

Comment:
can we simply use `bool` instead of `error`?

Reply:
That way you'll loose that informative `Failure` condition message:
`message: 'Error encountered while creating DataVolumes: DataVolume my-vol importer is not running due to an error: persistentvolumeclaims "my-vol" is forbidden: exceeded quota: storage, requested: requests.storage=10737418240, used: requests.storage=0, limited: requests.storage=2Gi'`

I think we should keep it.

Reply:
`Has...` usually expects a bool. You could simply return the condition and move the error message where the function is called